webpack.config.prod.tauri.js

```diff
@@ -0,0 +1,56 @@
+const webpack = require('webpack');
+const { merge } = require('webpack-merge');
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
+const CompressionWebpackPlugin = require('compression-webpack-plugin');
+const commonConfig = require('./webpack.config.common');
+
+module.exports = merge(commonConfig, {
```

Comment:
can production config be reused/merged?

Reply:
I agree that's possible
matter of time as lots of other things